common/views/components/PageLayout/index.tsx

```diff
@@ -325,7 +326,7 @@ const PageLayoutComponent: FunctionComponent<Props> = ({
               }}
             />
           )}
-        {popupDialog.data.isShown &&
+        {popupDialog?.data?.isShown &&
```

Comment:
Oops that's slid in there; Think we need this in case the popupDialog Prismic page gets unpublished. I have to test it.